### PR TITLE
Git attributes for linguist

### PR DESCRIPTION
**Pull request by kamre**
#### Summary
Git attributes for ignoring `website` and `swagger docs` in linguist added.

Currently this repo page shows this distribution of the languages:
image
which is a bit misleading because of auto generated documentation.

With the proposed changes it looks like this:
image

### CODE REVIEW

**Review by wiggin77 — APPROVED** [review]
LGTM :+1:   I agree @kamre this change makes the list more accurate.

**Comment by kamre** [comment]
Hi @jespino, @chenilim what do you think about this minor improvement?

**Comment by chenilim** [comment]
Super! Thanks so much @kamre! This makes sense. The "HTML" bar was throwing me off too. @wiggin77 not sure if you were waiting for more reviews, but you could have merged this after you approved it. I'll merge it now.

**Pull request merged**
